src/form.html

```
@@ -88,6 +121,27 @@ <h2>BackVault Setup</h2>
+                    <option value="single">Single combined file (requires raw encryption)</option>
+                </select>
+                <p class="org-error" id="org-error" hidden>Single mode is not supported with Bitwarden encryption. Use "Separate files" or set BACKUP_ENCRYPTION_MODE=raw.</p>
+                <script>
+                (function(){
+                    var select=document.getElementById('org_export_mode');
+                    var error=document.getElementById('org-error');
+                    select.addEventListener('change',function(){error.hidden=select.value!=='single'});
+                    error.hidden = select.value !== 'single';
+                })();
+                </script>
+                </script>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </script> ]

(tag-pair)